massa-final-state/src/final_state.rs

```diff
@@ -574,9 +576,39 @@ impl FinalState {
             &mut db_batch,
         );
 
+        let slot_ts = get_block_slot_timestamp(
+            self.config.thread_count,
+            self.config.t0,
+            self.config.genesis_timestamp,
+            slot,
+        )
+        .expect("Cannot get timestamp from slot");
+
+        let slot_prev_ts = get_block_slot_timestamp(
+            self.config.thread_count,
+            self.config.t0,
+            self.config.genesis_timestamp,
+            slot.get_prev_slot(self.config.thread_count)
+                .expect("Cannot get prev slot"),
```

Comment:
In tests, do we always attach to a Slot > Slot(0,0)?
I assume if the CI is fine with it, it's ok.

Reply:
I've run all the tests and it seems fine but maybe we can double check together?